cirq-google/cirq_google/engine/engine_processor_test.py

```diff
@@ -291,6 +291,7 @@ def test_supported_languages():
 
 
 def test_get_device_specification():
+    # TODO(verult)
     processor = cg.EngineProcessor('a', 'p', EngineContext(), _processor=quantum.QuantumProcessor())
     assert processor.get_device_specification() is None
```

Comment:
Why is this None?  Is this an incomplete test?

Reply:
This was here from the previous iteration of the test. I think this part just verifies that an unset DeviceSpecification doesn't break anything. The next section verifies an actual DeviceSpecification.